qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/VerifyLoginSteps.java

```diff
@@ -0,0 +1,67 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+import static org.junit.Assert.assertTrue;
```

Comment:
checkout the existing import order and align

Reply:
Resolved thnks